addon/core/consumer.js

```diff
@@ -19,7 +18,7 @@ export default EmberObject.extend({
 
   willDestroy() {
     this._super();
-    run(this.connection, 'destroy');
-    run(this.subscriptions, 'destroy');
+    this.connection.destroy();
```

Comment:
Harmless, but not unnecessary Ember.run call here.   I introduced those in my last PR.